catalog/USGS/USGS_NLCD_RELEASES_2019_REL_NALCMS.jsonnet

```diff
@@ -0,0 +1,463 @@
+    Canada selected most images from the year 2020 with a few from 2019 and
+    2021, the Conterminous United States employed mainly images from 2019, while
+    Alaska land cover maps are mainly based on the use of images from 2021. The
+    land cover map for Mexico was based on land cover change detection between
+    2015 and 2020 Mexico Landsat 8 mosaics.
+
+    In order to generate a seamless and consistent land cover map of North
+    America, national maps were generated for Canada by the CCRS; for Mexico by
+    CONABIO, INEGI, and CONAFOR; and for the United States by the USGS. Each
+    country chose their own approaches, ancillary data, and land cover mapping
+    methodologies to create national datasets. This North America dataset was
+    produced by combining the national land cover datasets.  The integration of
+    the three national products merged four Land Cover map sections, Alaska,
+    Canada, the conterminous United States and Mexico.
+
+    See also:
+
+    * Natural Resources Canada has North American Land Cover information
+      available online at
+      [https://open.canada.ca/data/en/dataset/ee1580ab-a23d-4f86-a09b-79763677eb47](
+      https://open.canada.ca/data/en/dataset/ee1580ab-a23d-4f86-a09b-79763677eb47)
+
+    * The National Commission for the Knowledge and Use of Biodiversity has
+      North American Land Cover information available online at
+      [https://www.biodiversidad.gob.mx/monitoreo/cobertura-suelo](https://www.biodiversidad.gob.mx/monitoreo/cobertura-suelo)
+
+    * The U.S. Geological Survey has North American Land Cover information
+      available online at [www.mrlc.gov](https://www.mrlc.gov)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    // 'land_cover',
+    // 'land_surface',
+    // 'land_use',
+    'landcover',
+    'landsat',
+    'nlcd',
+    // 'north_america',
+    'reflectance',
+    // 'remote_sensing',
+  ],
+  providers: [
+    ee.producer_provider(
+      'The Commission for Environmental Cooperation (CEC)',
+      'http://www.cec.org/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(
+      -175, 14, -50, 84,
+      '2019-01-01T00:00:00Z',
+      '2021-12-31T00:00:00Z'),
+  'gee:interval': {
+    type: 'cadence',
+    unit: 'year',
+    interval: 1,
+  },
+  summaries: {
+    gsd: [30],
+    'eo:bands': [
+      {
+        name: 'landcover',
+        // TODO(schwehr): Cleanup the formatting and then make gee:class
+        description: |||
+          Land cover classification grid cell value. Values are categorized into the following.
+          [Product Legend](https://www.mrlc.gov/sites/default/files/inline-images/2005_nalcms_large_0.jpg).
+          The classes in the 2020 product legend are given below.
+          |||,
+          'gee:classes': [
+            {
+              value: 1,
+              // TODO(schwehr): need to list the color for each or can it be left off? color: '466b9f',
```

Comment:
color is not required, but having the color is good.

Reply:
Got them updated to match the Legend exactly